.env.template

```diff
@@ -1,3 +1,7 @@
+DEMO_MODE=
+MVN_PROFILE=
```

Comment:
What is the difference between DEMO_MODE=true and MVN_PROFILE=demo? Do these configure different things?

Reply:
DEMO_MODE is no longer used.